### PR TITLE
[v14 - ready] Doc figure display performance update

### PR DESCRIPTION
**Pull request by TorkelE**
New version of https://github.com/SciML/Catalyst.jl/pull/950, this one everything builds locally without problems.

### CODE REVIEW

File: docs/src/model_creation/examples/basic_CRN_library.md [hunk 1]
```diff
@@ -115,12 +115,12 @@ using Plots
...
 splt = plot(ssol; title = "Chemical Langevin equation (SDE)")
 jplt = plot(jsol; title = "Stochastic chemical kinetics (Jump)")
-plot(oplt, splt, jplt; lw = 2, size=(800,800), layout = (3,1)) 
-plot!(bottom_margin = 3Plots.Measures.mm) # hide
-nothing # hide
+plot(oplt, splt, jplt; lw = 2, size=(800,800), layout = (3,1))
+oplt = plot(osol; title = "Reaction rate equation (ODE)", plotdensity = 1000, fmt = :png) # hide
+splt = plot(ssol; title = "Chemical Langevin equation (SDE)", plotdensity = 1000, fmt = :png) # hide
+jplt = plot(jsol; title = "Stochastic chemical kinetics (Jump)", plotdensity = 1000, fmt = :png) # hide
+plot(oplt, splt, jplt; lw = 2, size=(800,800), layout = (3,1), plotdensity = 1000, fmt = :png) # hide
```
Comment (TorkelE) [inline docs/src/model_creation/examples/basic_CRN_library.md:122]: Unless I add `plotdensity = 1000, fmt = :png` to the subplots the performance takes a sever hit.

File: docs/src/introduction_to_catalyst/introduction_to_catalyst.md [hunk 2]
```diff
@@ -196,6 +196,7 @@ jprob = JumpProblem(repressilator, dprob, Direct(), save_positions=(false,false)
 # now, let's solve and plot the jump process:
 sol = solve(jprob, SSAStepper(), saveat=10.)
 plot(sol)
+plot(sol, plotdensity = 1000, fmt = :png) # hide
```
Comment (isaacsas) [inline docs/src/introduction_to_catalyst/introduction_to_catalyst.md:199]: See my comments in the earlier pr.

**Review by isaacsas — CHANGES_REQUESTED** [review]
See my comments in the earlier pr. Fine to merge once they are addressed.

File: docs/src/introduction_to_catalyst/introduction_to_catalyst.md (hunk 2 above)
Comment (TorkelE) [reply docs/src/introduction_to_catalyst/introduction_to_catalyst.md:199]: done

**Pull request merged**